CharlieBackend.Api/Extensions/UserRoleExtension.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace CharlieBackend.Api.Extensions
+{
+    public static class UserRolesExtension
+    {
+        public static Roles ToEnum<Roles>(this string value)
+        {
+            return (Roles)Enum.Parse(typeof(Roles), value);
```

Comment:
What if [`Enum.Parse()`](https://docs.microsoft.com/en-us/dotnet/api/system.enum.parse?view=net-5.0#System_Enum_Parse_System_Type_System_String_) throws an exception?

Perhaps [`Enum.TryParse()`](https://docs.microsoft.com/en-us/dotnet/api/system.enum.tryparse?view=net-5.0#System_Enum_TryParse__1_System_String___0__) will be a more suitable choice.

Reply:
Not longer relevant since this extension method was removed completely.